src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {Component} from 'react';
```

Comment:
```suggestion
import React from 'react';
```
Let's get all from React namespace, e.g. `React.Component`

Reply:
Done